app/mailers/course_update_email_mailer.rb

```diff
@@ -0,0 +1,29 @@
+class CourseUpdateEmailMailer < GovukNotifyRails::Mailer
+  def course_update_email(course, attribute_changed)
```

Comment:
@SamJamCul I think this should take an argument of the user that is emailed - this will later be determined from the UserNotifications table (or whatever we call it) - I wouldn't expect the mailer to be doing the logic of deciding who to email

Reply:
Cool, can do.

Reply:
To expand on this:

The mailer itself should be responsible simply for sending out the email, in this case (given it's notify) that also includes adding the correct personalisations

I would expect the subsequent service that gets created to handle the logic of determining: 

- Who gets emailed
- The course to email about
- And what attribute is changed

As users will be able to turn it off and on by themselves